index/src/main.rs

```diff
@@ -39,14 +42,19 @@ fn main() {
     }
 }
 
+// TODO ask about try_main
 fn try_main() -> Result<()> {
     let opts = Opts::from_args();
     let repo = Repository::open(&opts.index)?;
     let crates = parse_index(&opts.index)?;
     let pb = setup_progress_bar(crates.len());
     let timestamps = compute_timestamps(repo, &pb)?;
     let crates = consolidate_crates(crates, timestamps);
-    write_json(crates)?;
+    let transitive = compute_transitive(&crates);
+
+    write_json(cargo_tally::JSONFILE, crates)?;
+    // or make a new function
+    write_json(cargo_tally::COMPFILE, transitive)?;
```

Comment:
Change `write_json` to accept `T: serde::Serialize` or make new `write_trans_json`